snap_http/api.py

```diff
@@ -355,3 +355,89 @@ def remove_user(username: str) -> SnapdResponse:
     """Remove a local user."""
     body = {"action": "remove", "username": username}
     return http.post("/users", body)
+
+
+# Apps and Services
+
+
+def get_apps(names: List[str] = [], services_only: bool = False):
+    """List available apps.
+
+    :param services_only: Return only services.
+    :param names: List apps for the snaps in `names` only.
+    """
+    query_params = {}
+
+    if services_only:
+        query_params["select"] = "service"
+
+    if names:
+        query_params["names"] = ",".join(names)
+
+    return http.get("/apps", query_params=query_params)
+
+
+def start(name: str, enable: bool = False) -> SnapdResponse:
+    """Start the service `name`.
+
+    :param enable: arranges to have the service start at system start.
+    """
```

Comment:
`start`, `stop`, and `restart` are just special-cases of their respective `_all` functions, so they could be defined more like

```python
    def start(name: str, enable: bool = False) -> SnapdResponse:
        return start_all([name], enable)
```

but I see that we haven't really set a precedence for doing things that way, and I don't want to get hamstrung if snapd decides to differentiate these calls, so I'm happy to leave it as-is.

Just wanted to record this for posterity somewhere.